### PR TITLE
[Chore] Bump morley dependency

### PR DESCRIPTION
**Pull request by lierdakil**
## Description

Problem: we need to check out how recent changes in morley
affect our packages, including morley-multisig and projects that
depend on it.

Solution: bump morley and affected dependencies to the latest
available (master) versions.

## Related issue(s)

Part of [TM-571](https://issues.serokell.io/issue/TM-571)

#### Related changes (conditional)

- Tests (see [short guidelines](../blob/master/CONTRIBUTING.md#tests))
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - I checked whether I should update the docs and did so if necessary:
    - [ ] [README](../blob/master/README.md)
    - [ ] Haddock
    - [ ] [docs/](../blob/master/docs/)
  - [ ] I updated [changelog](../blob/master/ChangeLog.md) unless I am sure my changes are
        really minor.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the following policy](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [x] My code complies with the [style guide](../blob/master/docs/code-style.md).

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -14,22 +14,25 @@ extra-deps:
...
+    - code/morley
     - code/morley-client
+    - code/morley-prelude
 
 - git:
     https://gitlab.com/morley-framework/morley-multisig.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    c2e041a8ab7ace0807e3c605e4bf60065cd6c9f1 # master
+    5e883bf46fce9c8bc6637576832745377b0a6f29 # bleeding edge
```
[the remark this answers is not in the document]
Comment (pasqu4le) [reply stack.yaml:29]: BTW some of these (`indigo` and `morley-upgradeable`) are already outdated, but I checked and they still compile with the latest commit too.

File: test/TestM.hs [hunk 2]
```diff
@@ -168,10 +167,23 @@ instance HasCmdLine TestM where
...
+getContractStorageAtBlock
+getContractBigMapAtBlock
+getBigMapValueAtBlock
+getBigMapValuesAtBlock
+getBalanceAtBlock
+getDelegateAtBlock
+runCodeAtBlock
+getChainId
+getManagerKeyAtBlock
+-}
```
[the remark this answers is not in the document]
Comment (lierdakil) [reply test/TestM.hs:181]: Oh, indeed.

File: stack.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (lierdakil) [reply stack.yaml:29]: FWIW, I've bumped to the latest. Also, apparently I've missed a few warnings the last time, fixed those too.

File: src/Client/Main.hs [hunk 3]
```diff
@@ -159,7 +159,7 @@ mainProgram cmd = case cmd of
     case pkgs of
       Left err -> printTextLn err
       Right packages -> runMultisigContract packages
-  CmdDeployContract (arg #owner -> mOwner) deployOptions -> do
+  CmdDeployContract (N mOwner :: "owner" :! Maybe AddressOrAlias) deployOptions -> do
```
Comment (pasqu4le) [inline src/Client/Main.hs]: I'd argue that the type signature is unnecessary here, but I'm also not particularly against it.

**Review by pasqu4le — APPROVED** [review]
All LGTM, I've pushed a small fixup only to update dependencies and for the nitpick below

**Pull request merged**